docs/source/SMG/workflow.rst

```diff
@@ -0,0 +1,65 @@
+Developer Workflow
+=====================================
+
+Workflow
```

Comment:
maybe link to maintenance section for summary of commands and useful URLs?

Reply:
Same here, I'll add it to the story.